### ISSUE TITLE
Unnecessary leading empty lines in index.html

### ISSUE THREAD

**Issue by pwwang** [mkdocs/mkdocs#2180]
Is there any reason to add 3 empty lines before `<!doctype HTML>` in index.html? It makes github pages not recognize it and report 404.

**Comment by waylan** [comment]
Which theme are you using? I don't see this behavior with either of the built in themes. If you are using a third party theme, please report this to that theme.

**Comment by pwwang** [comment]
I am using `material`. But I don't think it's related to any themes. I built the docs initially to GH pages, and it showed 404. I searched online and found a solution in this thread: https://github.community/t/index-html-not-working/10256/16. 

After I manually remove the empty lines, the page shows correctly.
It turns out that I have to add this kind of action (`fix-index`) in my workflow to make it work every time I rebuild the docs:

https://github.com/pwwang/liquidpy/blob/master/.github/workflows/docs.yml
